pycbc/distributions/constraints.py

```diff
@@ -60,7 +79,27 @@ def _constraint(self, params):
         return params[self.constraint_arg]
 
 
+class ConvexHull(Constraint):
```

Comment:
Call this class `SupernoveConvexHull` to be more explicit. Just calling it `ConvexHull` is a little too general.